CIP-XXXX/README.md

```diff
@@ -0,0 +1,377 @@
+> Cardano Blockchain budget through an on-chain governance action. No
+> withdrawals from the Cardano Blockchain treasury shall be permitted unless a
+> budget for the Cardano Blockchain is then in effect as required by the Cardano
+> Blockchain Guardrails. Cardano Blockchain budgets shall specify a process for
+> overseeing use of funds from Cardano Blockchain treasury withdrawals including
+> designating one or more administrators who shall be responsible for such
+> oversight.
+>
+> — Cardano (Draft) Constitution, Article III, Section 8
+
+> ### 3. GUARDRAILS AND GUIDELINES ON TREASURY WITHDRAWAL ACTIONS
+>
+> **Treasury withdrawal** actions specify the destination and amount of a number
+> of withdrawals from the Cardano treasury.
+>
+> #### Guardrails
+>
+> TREASURY-01 (x) DReps **must** define a net change limit for the Cardano
+> Treasury's balance per period of time.
+>
+> TREASURY-02 (x) The budget for the Cardano Treasury **must not** exceed the
+> net change limit for the Cardano Treasury's balance per period of time.
+>
+> TREASURY-03 (x) The budget for the Cardano Treasury **must** be denominated in
+> ada.
+>
+> TREASURY-04 (x) Treasury withdrawals **must not** be ratified until there is a
+> community approved Cardano budget then in effect pursuant to a previous
+> on-chain governance action agreed by the DReps with a threshold of greater
+> than 50% of the active voting stake.
+>
+> — Cardano (Draft) Constitution, Appendix I: Cardano Blockchain Guardrails),
+> Section 3
+
+
+While this may be satisfactory in general terms from a Constitutional
+perspective, it falls far short of a clearly defined set of prescriptive
+guardrails guiding the allocation, auditing, and custody of budgetary funds for
+both proposers, custodians, and community members to reference.
+
+This CIP aims to define the template for budget proposals specifically in
+relation to required and mandatory information that should be included in any
+proposed budget.
+
+## Specification
+
+> The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "
+> SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+> "OPTIONAL" in this document are to be interpreted as described
+> in [RFC 2119][RFC2119].
+
+> It is understood that 1 Ada is the equivalent of 1 Million (1 000 000)
+> Lovelace, the smallest, non-divisible unit of currency on the Cardano mainnet
+> blockchain.
+
+### Constitutional Requirements
+
+The following requirements are currently specified within the content of the
+Cardano (Draft) Constitution as specified in
+the [Motiviaton](#motivation-why-is-this-cip-necessary).
+
+> Changing or modifying these requirements would require a Constitution change.
+> This section **MUST** be updated if or when the verbiage of the
+> Constitution, specifically related to the budgetary process, is modified.
+
+* A budget for ongoing maintenance and future development **MUST** be proposed
+  once per year. _Budgets **MAY** be proposed more frequently._
```

Comment:
for clarity this should be rephrased as 
A budget for ongoing maintenance and future development **MUST** be proposed
  **at least once** per year.

Reply:
Agree with this one and there is another one that I missed that is part of the core Constitution so I will make this change along with that one! Thanks for spotting!